nwb_conversion_tools/datainterfaces/spikeglxdatainterface.py

```diff
@@ -94,15 +95,38 @@ def get_metadata(self):
                     description="The name of the ElectrodeGroup this electrode is a part of.",
                     data=shank_group_name
                 )
-            ],
-            ElectricalSeries_raw=dict(
-                name='ElectricalSeries',
+            ]
+        )
+
+        if self.source_data['file_path'].split('.')[-2] == 'ap':
```

Comment:
```suggestion
        if ".ap" in Path(self.source_data["file_path"]).suffixes:
```
I always prefer working with `pathlib` instead of string operations when dealing with file aspects. Simpler, cleaner, more explicit syntax.